tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -267,6 +268,7 @@ def test_positive_resource_vm_power_management(
             session.computeresource.vm_poweroff(name, rhev_data['vm_name'])
         else:
             session.computeresource.vm_poweron(name, rhev_data['vm_name'])
+        time.sleep(30)  # wait for the VM's status to change to powered on/off
```

Comment:
I think a retry loop with 1 sec delay will be a smarter idea. We'll be able to break just after the state change is detected. That way we only wait the necessary amount of time.

Reply:
I prefer to use `wait_for` library over retry loops